### PR TITLE
chore: convert response classes to data classes

### LINKED ISSUE #1595: convert susiresponse pojo class to data class
**Actual Behaviour**

Currently the response class is a simple pojo but since we use kotlin it should be a data class.

Parent issue #1568 

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by batbrain7**
Fixes #1595 

Changes:made response class to data class

### CODE REVIEW

File: app/src/main/java/org/fossasia/susi/ai/rest/responses/susi/SusiResponse.kt [hunk 1]
```diff
@@ -9,38 +9,38 @@ import java.util.ArrayList
...
  */
 
-class SusiResponse {
+data class SusiResponse(
 
-    @SerializedName("client_id")
-    @Expose
-    val clientId: String? = null
+        @SerializedName("client_id")
+        @Expose
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/susi/ai/rest/responses/susi/SusiResponse.kt]: No need of expose, these are public properties

File: app/src/main/java/org/fossasia/susi/ai/rest/responses/susi/SusiResponse.kt [hunk 2]
```diff
@@ -9,38 +9,38 @@ import java.util.ArrayList
...
+        val clientId: String? = null,
 
-    @SerializedName("query")
-    @Expose
-    val query: String = ""
+        @SerializedName("query")
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/susi/ai/rest/responses/susi/SusiResponse.kt]: No need of SerializedName on properties with same name

**Comment by arundhati24** [comment]
@batbrain7 Should #1568 not handle all of these files together? There is nothing unrelated in converting all these classes in the same package to data classes that a separate PR should be sent for each file.

**Comment by batbrain7** [comment]
@arundhati24 I am trying to contribute as much as I can, I am busy with my campus placements, so I am trying to do all that I can and the maximum that I can give to the project.

**Comment by arundhati24** [comment]
@batbrain7 This is a very basic change i.e. just adding a word, a few commas and changing the curly braces to  parentheses and that too in the same package and for same change. Please fix all of these together in one PR otherwise there would be unnecessary 31 PRs for this change, which IMHO is not required and would call for an unnecessary review process and even unnecessary extra time for sending so many PRs.

**Comment by batbrain7** [comment]
as said

> @arundhati24 I am trying to contribute as much as I can, I am busy with my campus placements, so I am trying to do all that I can and the maximum that I can give to the project.

**Comment by arundhati24** [comment]
Okay, if you're busy, please let me know, I will send a PR for this. This isn't a big change to do and can be done within 5-10 minutes.

**Comment by batbrain7** [comment]
@arundhati24 I will do this, if you want to work on issues then please integrate the feedback log api

**Comment by iamareebjamal** [comment]
@batbrain7 I know you are busy, as are other students. We can't have preferential treatment for specific students. Sending 5 PRs where one would suffice just to increase your contribution count will have more negative impact on your evaluation than sending respectable normal PRs as other students are sending

**Comment by batbrain7** [comment]
Okay @iamareebjamal this PR has been sent, is it ok that I make all the other additions in a new PR ?

**Comment by iamareebjamal** [comment]
No, please add all changes in one PR

We can't have 1 file PRs pollute the commit history

**Pull request merged**